comm/src/main/scala/coop/rchain/comm/transport/GrpcTransportServer.scala

```diff
@@ -84,7 +87,7 @@ class GrpcTransportServer[F[_]: Monixable: Concurrent: RPConfAsk: Log: Metrics](
             handleStreamed(blob)
         }) >> Metrics[F].incrementCounter("dispatched.packets")
 
-    for {
+    val cancelable = for {
```

Comment:
Here is the border to the code that still needs refactoring. GrpcTransportReceiver should also return Resource instead of Cancelable interface.